### PR TITLE
DEL-7301

### PR DESCRIPTION
**Pull request by Tej-Birring**
UI changes.

### CODE REVIEW

File: components/side-menu.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ function Menu() {
         <MenuElem
           item={item}
           key={item.path}
-          selected={router.asPath == item.path}
+          selected={router.asPath.startsWith(item.path)}
```
Comment (TudorCRL) [inline components/side-menu.tsx:83]: What's this change for?

File: components/common.tsx [hunk 2]
```diff
@@ -790,3 +791,55 @@ export const GetInTouchCalendlyBox = ({
...
+    () =>
+      breakVal
+        ? ({ children, ...props }) => <HStack {...props}>{children}</HStack>
+        : ({ children, ...props }) => <VStack {...props}>{children}</VStack>,
+    [breakVal]
+  );
+  const rootRef = useRef(null);
+  useEffect(() => {
+    rootRef.current = document.getElementById('__next');
+  }, []);
```
Comment (TudorCRL) [inline components/common.tsx:820]: Is this used anywhere?

File: components/code-examples.tsx [hunk 3]
```diff
@@ -1,115 +1,139 @@
...
-                token || `Ex4MPl370k3n`
-              }"`}
-            />
-            <DescriptionLabel pt='2em'>
-              To get output in JSON format, remove the format=txt query parameter from the GET
-              request.
+            <DescriptionLabel fontSize='1.1em'>
+              You can get started with our example{' '}
+              <a
+                href='https://docs.speechmatics.com/introduction/rt-guide#real-time-python-example/'
```
Comment (TudorCRL) [inline components/code-examples.tsx:124]: I assume this is tested

File: components/code-examples.tsx [hunk 4]
```diff
@@ -1,115 +1,139 @@
...
+              <DescriptionLabel pt='2em'>
+                To get output in JSON format, remove the format=txt query parameter from the GET
+                request.
+              </DescriptionLabel>
+            </Box>
+            <PanelDivider />
+            <DescriptionLabel mb='0'>
+              See our{' '}
+              <Link
+                href='https://docs.speechmatics.com/en/cloud/howto/'
```
Comment (TudorCRL) [inline components/code-examples.tsx:110]: I asumme this is tested

Comment (Tej-Birring) [reply components/code-examples.tsx:110]: Only manually.

File: components/code-examples.tsx (hunk 3 above)
Comment (Tej-Birring) [reply components/code-examples.tsx:124]: As above.

File: components/side-menu.tsx (hunk 1 above)
Comment (Tej-Birring) [reply components/side-menu.tsx:83]: Highlighting of the side menu item.

File: components/common.tsx (hunk 2 above)
Comment (Tej-Birring) [reply components/common.tsx:820]: Yes, it is being used in: batch-demo and real-time-demo

Comment (TudorCRL) [reply components/common.tsx:820]: I mean, the rootRef object. Can't tell if I'm just missing something but I can't see it being used anywhere

File: components/side-menu.tsx (hunk 1 above)
Comment (Tej-Birring) [reply components/side-menu.tsx:83]: The change originates from @jameso-speechmatics' commit:
"update active link logic to include queries"

I think that's a fair description. With a simple equality check, the side menu item is not highlighted when a query is passed to the route.

Comment (TudorCRL) [reply components/side-menu.tsx:83]: Cool, makes sense

File: components/common.tsx (hunk 2 above)
Comment (Tej-Birring) [reply components/common.tsx:820]: Looks like it is being used in GetInTouchCalendlyBox' render funct.

Comment (TudorCRL) [reply components/common.tsx:820]: Okay, in that case we can remove it I believe. It's not being used in this function as this function just renders a button with a link

**Review by TudorCRL — APPROVED** [review]

**Pull request closed without merging**